examples/react/getting-started/src/config/index.js

```diff
@@ -0,0 +1,11 @@
+require('dotenv').config();
```

Comment:
@wjohnsto So since the Express server to enable previews is in JavaScript we would need to setup a transpiler. I figure for an example app, that may be too much boilerplate